### PR TITLE
Refactor tests

### PR DESCRIPTION
**Pull request by yalef**
Use drf client instead of django client in api tests module

### CODE REVIEW

File: tests/test_api/test_export.py [hunk 1]
```diff
@@ -1,41 +1,38 @@
-from django.contrib.auth.models import User
-from django.test.client import Client
 from django.urls import reverse
 
+from rest_framework import status
+from rest_framework.test import APIClient
```
Comment (TheSuperiorStanislav) [inline tests/test_api/test_export.py:4]:
```suggestion
from rest_framework import status, test
```

And then test.APIClient

**Review by TheSuperiorStanislav — APPROVED** [review]

**Comment by TheSuperiorStanislav** [comment]
Also django 5.0 released, maybe add it to matrix?

**Comment by yalef** [comment]
> Also django 5.0 released, maybe add it to matrix?

Great idea! Added in #26 since we start changing testing matrix there

File: tests/test_api/test_export.py [hunk 2]
```diff
@@ -1,41 +1,37 @@
...
-
-    response = client.post(
+    response = admin_api_client.post(
         path=reverse("export-artist-start"),
         data={
             "file_format": "csv",
         },
     )
+    assert response.status_code == status.HTTP_201_CREATED
     assert response.data["export_status"] == "CREATED"
```
Comment (OttoAndrey) [inline tests/test_api/test_export.py:20]: Maybe use `CREATED` from `ExportStatus` choices?

File: tests/test_api/test_export.py [hunk 3]
```diff
@@ -1,41 +1,37 @@
...
     assert ExportJob.objects.first()
```
Comment (OttoAndrey) [inline tests/test_api/test_export.py:21]: Maybe use `id` from response and then `assert ExportJob.objects.filter(id=ID).exists()?`

**Review by OttoAndrey — APPROVED** [review]

**Pull request merged**